docs/docs/start-building/other-languages.mdx

```diff
@@ -15,7 +15,7 @@ programming languages:
 - [JavaScript](https://www.npmjs.com/package/@ory/client) with TypeScript
   definitions and compatible with: NodeJS, ReactJS, AngularJS, Vue.js, and many
   more.
-- [PHP](https://packagist.org/packages/ory/client)
```

Comment:
Thank you! The URL *should* be correct but it looks like there is an issue in publishing the SDK :/

Reply:
Right. 
But then it should use a wildcard
https://packagist.org/search/?query=ory%2F%2Aclient

ory/*client